pkg/ccl/changefeedccl/batching_sink.go

```diff
@@ -101,12 +101,14 @@ type flushReq struct {
 // but separate from the encoded keys and values.
 type attributes struct {
 	tableName string
+	headers   map[string][]byte
```

Comment:
Can we define map[string][]byte (or however we decide to represent headers) as its own type?